lib/src/frame.dart

```diff
@@ -56,6 +80,42 @@ final _firefoxSafariFrame = RegExp(r'^'
     // empty in Safari if it's unknown.
     r'$');
 
+// With names:
+//
+// g@http://localhost:8080/test.wasm:wasm-function[796]:0x143b4
+// f@http://localhost:8080/test.wasm:wasm-function[795]:0x143a8
+// main@http://localhost:8080/test.wasm:wasm-function[792]:0x14390
+//
+// Without names:
+//
+// @http://localhost:8080/test.wasm:wasm-function[796]:0x143b4
+// @http://localhost:8080/test.wasm:wasm-function[795]:0x143a8
+// @http://localhost:8080/test.wasm:wasm-function[792]:0x14390
+//
+// Matches named groups:
+//
+// - "member": Function name, may be empty: `g`.
+// - "uri": `http://localhost:8080/test.wasm`.
+// - "index": `796`.
+// - "offset": (in hex) `143b4`.
+final _firefoxWasmFrame = RegExp(r'^(?<member>.*)@(?:(?<uri>.*):wasm-function'
+    r'\[(?<index>\d+)\]:0x(?<offset>[0-9 a-f A-F]+))$');
+
+// With names:
+//
+// <?>.wasm-function[g]@[wasm code]
+// <?>.wasm-function[f]@[wasm code]
+// <?>.wasm-function[main]@[wasm code]
+//
+// Without names:
+//
+// <?>.wasm-function[796]@[wasm code]
+// <?>.wasm-function[795]@[wasm code]
+// <?>.wasm-function[792]@[wasm code]
+//
+// Matches named group "member": `g` or `796`.
+final _safariWasmFrame = RegExp(r'^.*\[(?<member>.*)\]@\[wasm code\]$');
```

Comment:
I'd consider matching as much of the known format as possible, to not accidentally matching another platform's stack trace. (Although `[wasm code]` makes that a little harder.)

Maybe:
```dart
final _safariWasmFrame = RegExp(
 r'^<\?>\.wasm-function\[(?<membe>[^\]]+)\]@\[wasm code\]');
```